### ISSUE TITLE
How to use gradient accumulation and adjust accumulation step in training scripts?

### ISSUE THREAD

**Issue by srzer** [THUDM/slime#1126]
Dear Slime team, 
I found that there seemed no guidance on setting gradient accumulation in the instruction, so I wonder whether it is feasible and am writing for your help.
Thanks!
